File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var answer = 0;
    let width = []; // 가로
    let height = []; // 세로
    
    for(let i = 0 ; i < sizes.length ; i++){
        let w = Math.max(sizes[i][0],sizes[i][1]);
        let h = Math.min(sizes[i][0],sizes[i][1]);
        width.push(w);
        height.push(h);
    }
    
    
    return Math.max(...width) * Math.max(...height);
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
public class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int max = 0; int min = 0;
        for(int i = 0 ; i < sizes.length; i++){
            max = Math.max(max,Math.max(sizes[i][0],sizes[i][1]));
            min = Math.max(min,Math.min(sizes[i][0],sizes[i][1]));
        }
        return max * min;
    }

    public static void main(String[] args) {
        Solution sol = new Solution();
        int[][] sizes = {{60,50},{30,70},{60,30},{80,40}};
        System.out.println(sol.solution(sizes));
    }
}
